blog/_posts/2017-09-8-railsgyn-lastblogpost.md

```diff
@@ -0,0 +1,59 @@
+---
+title: RailsGyn come back to reality.
+layout: post
+created_at: Fri September 22 2017
```

Comment:
Hi, thanks for the PR. Your blogpost is scheduled to be published on Thursday September 27, 2017. Can you please update the date everywhere?